src/Microsoft.AspNetCore.Authentication.Abstractions/ISecureDataFormat.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    public interface ISecureDataFormat<TData>
```

Comment:
It's only used on Options types, right? Then it's fine to move to the concrete package.